kp_roll: 1
ki_roll: 0
kd_roll: 1
kp_pitch: 1
ki_pitch: 0
kd_pitch: 0.6
kp_yaw: 1
ki_yaw: 0
kd_yaw: 0.6
kp_x: 1
ki_x: 0
kd_x: 0.5
kp_y: 1
ki_y: 0
kd_y: 0.5
kp_z: 2
ki_z: 0
kd_z: 0.5
speed_coeff: 0.03
